apps/meteor/app/autotranslate/server/autotranslate.ts

```diff
@@ -165,7 +165,7 @@ export abstract class AutoTranslate {
 	tokenizeURLs(message: IMessage): IMessage {
 		let count = message.tokens?.length || 0;
 
-		const schemes = settings.get<string>('Markdown_SupportSchemesForLink')?.split(',').join('|');
+		const schemes = 'http,https';
```

Comment:
Note: I'll be adding some context here of why I made some decisions

- This `http,https` is the default setting from this setting, and this function is being used inside the `tokenize` function that is referenced inside `translateMessage`

Reply:
Maybe it would be better to check if it's an URL in a different way, because we make an URL withou http too, and this could be translated... 

Check `isValidLink`

Reply:
Looks like this is not being translated even without HTTP, so it might be good to go.